resolwe_bio/descriptors/reads_template.yml

```diff
@@ -0,0 +1,13 @@
+- name: reads_annotation
+  label: Reads annotation
+  description: Annotation of raw sequencing data
+  var:
```

Comment:
Ok, I will join `static` and `var` dicts and put them together into `schema` for backward compatibility. And from now on we should call this field `schema` also in files.

So @jkokosar please rename `var` to `schema`.